.github/workflows/buid_and_release.yml

```diff
@@ -39,3 +39,5 @@ jobs:
           output/release/medgen.obo
           output/release/medgen-disease-extract.obo
           output/release/medgen-xrefs.robot.template.tsv
+          output/release/hpo-umls.sssom.tsv
+          output/release/hpo-mesh.sssom.tsv
```

Comment:
## Any of this HPO data useful?
- [ ] Review @matentzn @twhetzel 

I found this file along with my other FTP files: [MedGen_HPO_Mapping.txt.gz](https://github.com/monarch-initiative/medgen/files/14736890/MedGen_HPO_Mapping.txt.gz)

I don't need this particular file for HPO<->MedGen mappings. I'm using their standard mapping file `MedGenIDMappings.txt`. I assume it has the same mappings. But I see some additional fields here. One thing I could do would be to take the HPO label so that my `hpo-umls.sssom.tsv` could have `object_label` in addition to `subject_label`. Otherwise, IDK how I would exactly use any of these other columns. Here's some sample rows:

|#CUI|SDUI|HpoStr|MedGenStr|MedGenStr_SAB|STY|
|-|-|-|-|-|-|
|C4025901|HP:0000002|Abnormality of body height|Abnormality of body height|GTR|Finding|
|C3714581|HP:0000003|Multicystic kidney dysplasia|Multicystic kidney dysplasia|GTR|Disease or Syndrome|
|C1708511|HP:0000005|Mode of inheritance|Mode of inheritance|GTR|Genetic Function|
|C0443147|HP:0000006|Autosomal dominant inheritance|Autosomal dominant inheritance|GTR|Intellectual Product|

Regarding the "STY" column, I've seen this acronym before but I can't find what it means. Do you guys know?

Reply:
STY -> Semantic Type (UMLS)

Cant tell you about the source, use your judgement or check with Megan!